### ISSUE TITLE
Fake code

### ISSUE THREAD

**Issue by Nitsugua38** [neckfire/website#1]
Looks like it’s just some random pieces of fake code

**Comment by neckfire** [comment]
What are you talking About

**Comment by Nitsugua38** [comment]
bahaha u know what i mean
u don’t exist
u r a jk
arpon os is fake, ur musics are not made by u
u just fake bro !

**Comment by neckfire** [comment]
Haha, i just teached everything u know, if you want to talk with me, propose something, else you can return inside the battlebus or in hypixel...
P.S: my arpon os is waiting for an approbation of the linux Foundation,  so go back inside your junk.

**Comment by Nitsugua38** [comment]
oh, what did you teach me then? English? I went in Ireland! JavaScript? Learned all by myself, MDN and StackOverflow! HTML? Same! Discord? Knew it before you! GitHub? Same! Linux? First OS I ever used!
u didn’t teach me nothing bro!

I propose you the following: learn how to code instead of copy/pasting random things.

Ur repos are either empty or contain random files that make no sense!

Now, check my repo: https://github.com/Nitsugua38/Nitsubot-Revolutionary-Multi-Purposes/
This my Verified Discord Bot actually used in 102 servers! Here are the real and concrete projects! And everything is FROM ME! Just google it, you’ll find my code nowhere else caus I made it! And ppl are so happy, they say: "I found a bot that can do everything for free and that works better than the others!"

Now, I have downloaded your "arponos" and it is just…funny, I know how an ISO works and what it should includes and urs is just missing those files, it’s just a fake thing, one more time!
And u’ve added "my arpon os is waiting for an approbation of the linux Foundation": OMG THAT SOUNDS LIKE THE BIGGEST SCAM I HAVE EVER HEARD! bahahahaha
aight, talk to me again when u’ll have acquired some IT and CS skills
